src/openrct2/park/ParkPreview.cpp

```diff
@@ -108,32 +115,58 @@ namespace OpenRCT2
             }
         } while (!(tileElement++)->IsLastForTile());
 
+        // Darken every other tile that's outside of the park, unless it's a path
+        if (isOutsidePark && _tileColourIndex == 1 && colour != PALETTE_INDEX_17)
```

Comment:
Could you rename `colour` to `paletteIndex`? `colour` implies one of the `COLOUR_` constants (and in a future refactor, they’ll both be strong enums that are incompatible anyway).